app/javascript/components/DownloadButton.js

```diff
@@ -108,17 +110,35 @@ function DownloadCommandContainer(props) {
  */
 export default function DownloadButton(props) {
 
+  const searchContext = useStudySearchContext()
```

Comment:
this is an interesting pattern I haven't seen before.  I could see renaming the method to useContextStudySearch just so that it's really clear that it's a useContext hook and we can be careful of all the usual hook concerns, but I'll defer to you.

Reply:
Good readability suggestion, done in b72b6048.

This pattern enables testing components that use `useContext` in Jest / Enzyme.  It builds atop the [approach described here](https://stackoverflow.com/a/56380444).  It works, but feels kludgy.